cognite/neat/_client/_api/schema.py

```diff
@@ -126,3 +127,28 @@ def get_hierarchical_properties(
                 if is_hierarchy_property(prop.container, prop.container_property_identifier):
                     hierarchical_properties_by_view_id[view.as_id()].add(prop_id)
         return hierarchical_properties_by_view_id
+
+    @staticmethod
+    def get_view_order_by_direct_relation_constraints(views: Iterable[dm.View]) -> list[dm.ViewId]:
+        """Sorts the views by container constraints."""
+        view_sequence = list(views)
+        view_ids_by_container: dict[dm.ContainerId, set[dm.ViewId]] = defaultdict(set)
+        for view in view_sequence:
+            for container_id in view.referenced_containers():
+                view_ids_by_container[container_id].add(view.as_id())
+
+        view_by_dependency: dict[dm.ViewId, set[dm.ViewId]] = {}
+        for view in view_sequence:
+            view_id = view.as_id()
+            view_by_dependency[view_id] = set()
+            for prop in view.properties.values():
+                if (
+                    isinstance(prop, dm.MappedProperty)
+                    and isinstance(prop.type, dm.DirectRelation)
+                    and prop.type.container
+                ):
+                    view_by_dependency[view_id].update(view_ids_by_container[prop.type.container])
+        try:
+            return list(TopologicalSorter(view_by_dependency).static_order())
```

Comment:
We need to create nodes in views considering the direct relations constraints. Typically, it is used to require CogniteSourceSystem nodes are created before pointing to them.